app/views/core/RootView.coffee

```diff
@@ -252,6 +258,25 @@ module.exports = class RootView extends CocoView
         legacyTitle: legacyTitle
       }
     })
+  
+  # Attach the navigation Vue component to the page
+  initializeNavigation: ->
+    staticNav = document.querySelector('#main-nav')
+
+    if @navigation and staticNav
+      staticNav.replaceWith(@navigation.$el)
+      return
+
+    return unless staticNav
+
+    @navigation = new Navigation({
+      el: staticNav
+    })
+
+    # Hack - It would be better for the Navigation component to manage the language dropdown.
+    setTimeout(() =>
+      @buildLanguages()
+    , 0)
```

Comment:
More CoffeeScripty would be to not use a `()` here that we don't need, and `_.defer` is more commonly used instead of `setTimeout` with 0. Note we don't call it if we have been destroyed in the interim frame to avoid errors:
```suggestion
    _.defer => @buildLanguages?()
```